.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -11,5 +11,9 @@ assignees: ''
 ----------------------------------------------------------------------------
 After deleting this template, please provide the following information:
 * Forge Version
-* *If updating was not possible*; Building Gadgets version
-* *If the issue is a crash*; An external link to the crash report
+* Building Gadgets version 
+  * Use the version Forge shows you *In-Game*
+* *If the issue is a crash*; An external link to the crash report and the **action caused crash** (e.g. update mod)
```

Comment:
shouldn't it be 
```suggestion
* *If the issue is a crash*; An external link to the crash report and the **action that caused the crash**
```
What do you mean with "e.g. update mod"? In theory updating the mod should not be the reason for the crash. And even if it only happens in a newer version, it won't help us much if people say "here's the crashreport and it happend after I updated the mod"